### PR TITLE
Created todo.json

### PR DESCRIPTION
**Pull request by qbebo**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/todoo.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "To publish my todo list tool with ur nice subdomain",
+    "repo": "https://iibeboq.github.io/todo/",
+    "owner": {
+        "username": "iibeboq",
+        "email": "hi@beboq.xyz",
+        "twitter": "ce3s"
+    },
+    "record": {
+        "CNAME": "iibeboq.github.io/todo"
```
Comment (WilliamDavidHarrison) [inline domains/todoo.json:10]: Please remove the `/todo`.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/todoo.json (hunk 1 above)
Comment (iibeboQ) [reply domains/todoo.json:10]: On repo or cname?

File: domains/todoo.json [hunk 2]
```diff
@@ -7,6 +7,6 @@
         "twitter": "ce3s"
     },
     "record": {
-        "CNAME": "iibeboq.github.io/todo"
+        "CNAME": "iibeboq.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/todoo.json:10]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/todoo.json (hunk 2 above)
Comment (iibeboQ) [reply domains/todoo.json:10]: > 404?

Bc its with /todo not iibeboq.github.io

Comment (WilliamDavidHarrison) [reply domains/todoo.json:10]: oh

**Review by wdhdev — APPROVED** [review]

File: domains/todoo.json (hunk 2 above)
Comment (iibeboQ) [reply domains/todoo.json:10]: > oh

Now i will do anything again? Make it back to /todo?

Comment (WilliamDavidHarrison) [reply domains/todoo.json:10]: Keep it like this. It will work even though the repo is called `todo`.

**Pull request closed without merging**